routes/actions.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+
+const actions = require("../data/helpers/actionModel");
+
+const router = express.Router();
+
+router.get("/", (req, res) => {
+  actions
+    .get()
+    .then(foundActions => {
+      res.json(foundActions);
+    })
+    .catch(err => {
+      res.status(404).json({ message: "no access" });
+    });
+});
+
+router.post("/", async (req, res) => {
+  try {
+    const action = await actions.insert(req.body);
+    res.status(201).json(action);
+  } catch (error) {
+    console.log(error);
+    res.status(500).json({
+      message: "No project with that id"
+    });
+  }
+});
+
+router.delete("/:id", (req, res) => {
+  const { id } = req.params;
+  actions
+    .remove(id)
+    .then(actionRemoved => {
+      if (actionRemoved === 0) {
+        res.status(404).json({ message: "no access" });
+      } else {
+        res.json({ success: "Action Removed" });
+      }
+    })
+    .catch(err => {
+      res.status(500).json({ message: "no access" });
+    });
+});
+
+router.put("/:id", async (req, res) => {
+  const { project_id, description, notes, completed } = req.body
+  const { id } = req.params;
+
+  if (!actions.get(req.params.id)) {
```

Comment:
This is really, really good. Great job nesting your db helper functions to get the status results you were wanting.